Automattic/newspack-blocks
# Carousel Block

### All Submissions:






### Changes proposed in this Pull Request:



Adds a Carousel block that uses `amp-carousel` to display posts.  `amp-carousel` cannot currently be used in the editor because it does not support DOM mutation after page load, which occurs whenever the array of posts to be displayed changes. Eventually Bento AMP will resolve this, but until this is available we will use the Swiper library to make the block functional in the editor. Currently, there is only a placeholder in the editor.

<img width="857" alt="Screen Shot 2019-10-08 at 12 18 29 AM" src="https://user-images.githubusercontent.com/1477002/66367198-344be780-e961-11e9-818e-b2d64c8f3d4a.png">

Closes https://github.com/Automattic/newspack-blocks/issues/48.

### How to test the changes in this Pull Request:

1. Check out the branch and execute `npm run build:webpack`
2. Add Articles Carousel block to a post.

### Other information:
